### PR TITLE
Fix PDF doc builds

### PR DESCRIPTION
**Pull request by samjwu**
RE: Error: Failed to load PDF document
image

Specify font package for xelatex engine for pdf
Ref: https://www.sphinx-doc.org/en/master/latex.html#module-latex

Specify image size for svg

### CODE REVIEW

**Review by Rmalavally — DISMISSED** [review]

**Review by saadrahim — CHANGES_REQUESTED** [review]
I think there are formatting and potentially linting issues with this. Please check again.

File: docs/conf.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
 # Keep capitalization due to similar linking on GitHub's markdown preview.
 shutil.copy2('../CHANGELOG.md','./CHANGELOG.md')
 
-latex_engine = "xel  atex"
+latex_engine = "xelatex"
```
Comment (Rmalavally) [inline docs/conf.py:17]: @samjwu We should add this word to the wordlist

**Comment by samjwu** [comment]
redone in https://github.com/RadeonOpenCompute/ROCm/pull/2486

File: docs/conf.py (hunk 1 above)
Comment (samjwu) [reply docs/conf.py]: this is in conf.py, which is not checked

Comment (Rmalavally) [reply docs/conf.py]: Sorry, this comment wasn't for this PR. Will delete. 

Thanks for clarifying, Sam.

**Pull request closed without merging**